hledger-mode.el

```diff
@@ -53,6 +53,11 @@
   :type 'face
   :group 'hledger)
 
+(defcustom hledger-account-colon-face hledger-account-face
```

Comment:
We can probably use `defface` here directly. When I wrote the other `defcustom` variables for faces in this file, I wasn't probably familiar with defface and how it's like `defcustom` but specifically for faces.

Reply:
I thought that might be the case! I’ll change it accordingly.